index.html

```diff
@@ -1,8 +1,153 @@
+      <figure>
+         <img src="images\PEBBLES1.JPG" width="400" height="300">
+         <figcaption>Pebbles, the adventurer.</figcaption>
+      </figure>
+
+      <figure>
+         <img src="images\PEBBLES2.JPG"width="400" height="300">
+         <figcaption>Pebbles, the coding kitty.</figcaption>
+      </figure>
+      </article>
+      
+      <center><br><a href = #top>Back to Top</a></center><br>
+          
+      </main>
+
+      <main>
+      <h1>Dogs</h1>
+
+      <article id = "bud">
+      <h2>Bud</h2>
+      <p>Bud (legal name, Budweiser&reg;) was technically not my dog but he was an integral part of my childhood. Bud belonged to my sister; he was a birthday present from her husband after Bud had been found abandoned near their house. Bud absolutely <b>loved</b> beer. Specifically, he loved Budweiser beer - hence, his name. When my sister divorced, Bud came to live with my parents and me (plus <a href=#callieholly>Callie &amp; Holly</a>). Bud and the cats tolerated each other well enough. My family thoroughly believes, to this day, that Bud was part human. Bud also loved popsicles in the summer, snuggling behind my mom on her favorite chair, and taking naps with me on the couch. Much to my dismay, when my sister and her husband remarried, Bud went back to live with them and his rottweiler brother, Yukon.</p>
+      <figure>
+         <img src="images\BUD.jpg" width="400" height="300">
+         <figcaption>Bud, loving life.</figcaption>
+         </figure>
+         </article>
+
+      <center><br><a href = #top>Back to Top</a></center>
+      
+      <article id = "malibu">
+      <h2>Malibu</h2>
+      <p>The greatest emotional attachment I ever formed to an animal was my baby Malibu. In 2003, I was diagnosed with a cyst in my brain and, after a few months of uncertainty surrounding my MRI, was told that my condition was not going to require surgery. After the confirming appointment at BJC, my parents and I went to Chesterfield Mall and found the pet store was having a sale on puppies. This is where we met Malibu. Malibu was the kindest, most gentle dog I have ever loved. She was easily trained (quite a feat for a beagle), loved people, and helped guide me through some of the most difficult times in my life. Malibu was also very polite; one of her commands was to say "please" before receiving a treat and she would let out a gentle "woof". In 2014, Malibu's heart, which had loved so much and so unconditionally, gave out and she crossed the rainbow bridge with me at her side, telling her how much I loved her. I will forever miss her and I keep some of her ashes in a nip-sized bottle of Malibu rum (after which she was named).</p>
```

Comment:
When it comes to paragraphs like this, for readability, I recommend hitting enter after every sentence or so, so you don't have such a long single line where you need to scroll over so much. Don't worry, this won't change how it appears on your webpage, but just makes the code a little more readable!